content/en/docs/ops/prep/deployment/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Deployment Best Practices
+description: General best practices for your Istio deployments.
+weight: 2
+icon: best-practices
+keywords: [deployment-models, cluster, availability-zones, control-plane]
+---
+
+We have identified the following general principles to help you get the most
+out of your Istio deployments. These best practices aim to limit the impact of
+bad configuration changes and make managing your deployments easier.
+
+## Prefer simpler deployment models
```

Comment:
@frankbu This section is meant to point folks towards the deployment models for them to choose the simplest deployment that suits their needs. The idea of having the simplest deployment might be obvious, but what the options are and where to find them are not. I agree that further instruction on how to chose would be great but that is something we can add later.

Reply:
Yeah, I understand the direction you want to go with this item, but until it says something specific about how to know what's simiplest, it sounds like silly obvious advice, and best to leave it out for now IMO. Reading it, as is, it's just obvious advice for any design. What's the alternative to following this best practice - make your deployment more complex than necessary.

Reply:
I disagree strongly because it is obvious to us what a complex vs a simple deployment looks like but that's only because of the curse of knowledge. I've taken in out for the sake of expedience though.

Reply:
I don't understand what you mean. I'm saying the users that is reading this will not have any idea what to do to follow this "use the simplest deployment" advice, because they don't have the "curse of knowledge". That said, trying to keep it simple is obvious advice, so we need more information about how to do that and not just state the obvious part.